clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/cache/RedisCacheConfig.groovy

```diff
@@ -47,7 +47,7 @@ class RedisCacheConfig {
     JedisSource jedisSource,
     ObjectMapper objectMapper,
     @Value('${redis.maxMsetSize:250000}') int maxMsetSize) {
-    new RedisNamedCacheFactory(jedisSource, objectMapper, maxMsetSize)
+    new RedisNamedCacheFactory(jedisSource, objectMapper, maxMsetSize, null)
```

Comment:
I was logging it at first (it was skipping a whole ton of items) but I went the `CacheMetrics` route because it let me actually confirm via a test case and it opens the door to wiring something up to export the metrics out via Spectator if we feel like we really need them.